.github/actions/spelling/reject.txt

```diff
@@ -1,6 +1,7 @@
 ^attache$
 ^attacher$
 ^attachers$
+^bellow$
```

Comment:
another archaic word I ran across... -- it's unlikely you'll want to reference stoking fires quite like this.

Reply:
UGH THANK YOU